Frameworks/UIKit/UIButton.mm

```diff
@@ -226,15 +226,6 @@ - (void)_initUIButton {
                                               e.handled = YES;
                                               [weakSelf _processPointerCaptureLostCallback:sender eventArgs:e];
                                           });
-
-    XamlControls::HookLayoutEvent(_xamlButton, ^(RTObject*, WUXIPointerRoutedEventArgs*) {
-        // Since we are using XAML Button behind the scene, the intrinsicContentSize calculation is done
-        // by XAML.
-        // The size of XAML elements(for eg Image) is calculated at runtime and then the
-        // intrinsicContentSize is invalidated.
```

Comment:
Correct; this isn't needed.  We handle image loading on this side and trigger layout as needed.

---
In reply to: [106746566](https://github.com/Microsoft/WinObjC/pull/2265#discussion_r106746566) [](ancestors = 106746566)